docs/pages/database-access/guides/postgres-self-hosted.mdx

```diff
@@ -156,6 +156,14 @@ $ tsh db logout example-postgres
 $ tsh db logout
 ```
 
+## Reference
```

Comment:
I don't think we need a whole "Reference" section for this information, since the table only has one row. I would explain the counterintuitive behavior in a paragraph where we show the user how to connect to Postgres through Teleport. This also makes the paragraph more visible.